modules/install/pages/upgrade.adoc

```diff
@@ -97,16 +101,16 @@ For example, if you are upgrading three enterprise nodes (`*Node{nbsp}1*`, `*Nod
 | Starting Version | Path to Current Version
 
 | 5.x
-| 5.x -> 6.0 -> 6.5 -> 7.2x
+| 5.x -> 6.0 -> 6.6 -> 7.2.x (except 7.2.3) -> 7.2.3
 
 | 6.x
-| 6.0 -> 6.5 -> 7.2x
+| 6.0 -> 6.6 -> 7.2.x (except 7.2.3) -> 7.2.3
 
 | 7.x
-| 7.0 -> 7.1 -> 7.2x
+| 7.0 -> 7.1 -> 7.2.x (except 7.2.3) -> 7.2.3
```

Comment:
When upgrading from 7.0 it is not necessary to go via 7.1. As well as it is not necessary to go via 7.2.x if you are already 7.1. So I think it should be changed to two options (same as for enterpise):

```
7.0 -> 7.1 -> 7.2.3
7.0 -> 7.2.x (except 7.2.3) -> 7.2.3
```